config/rbac/role.yaml

```diff
@@ -16,6 +16,7 @@ rules:
   - pods
   - serviceaccounts
   - services
+  - workflows
```

Comment:
Not needed here!

Reply:
55f476275436a9db1e5a394a7b1771076e1a4cfc